redbot/core/dev_commands.py

```diff
@@ -336,17 +336,33 @@ async def mock(self, ctx, user: discord.Member, *, command):
 
     @commands.command(name="mockmsg")
     @checks.is_owner()
-    async def mock_msg(self, ctx, user: discord.Member, *, content: str):
+    async def mock_msg(self, ctx, user: Union[discord.Member, discord.User, int], *, content: str):
```

Comment:
Why should we accept a User in this command?

Reply:
yeah i removed that